src/kvilib/locale/KviLocale.cpp

```diff
@@ -477,26 +477,22 @@ KviLocale::KviLocale(QApplication * pApp, const QString & szLocaleDir, const QSt
 	QString szLangFile = QString("%1/%2").arg(szForceLocaleDir, KVI_FORCE_LOCALE_FILE_NAME);
 
 	if(KviFileUtils::fileExists(szLangFile))
-	{
-		QString szTmp;
-		KviFileUtils::readFile(szLangFile, szTmp);
-		g_szLang = szTmp;
-	}
+		KviFileUtils::readFile(szLangFile, g_szLang);
 	if(g_szLang.isEmpty())
-		g_szLang = KviEnvironment::getVariable("KVIRC_LANG");
+		g_szLang = KviEnvironment::getVariable(QString("KVIRC_LANG"));
 #ifdef COMPILE_KDE_SUPPORT
 	if(g_szLang.isEmpty())
-		g_szLang = KviEnvironment::getVariable("KDE_LANG");
+		g_szLang = KviEnvironment::getVariable(QString("KDE_LANG"));
```

Comment:
QString is implicitly convertible from const char*, you don't need to add QString()